va/va_fei_h264.h

```diff
@@ -318,6 +318,115 @@ typedef struct _VAEncFEIDistortionH264 {
     uint32_t    reserved1[2];
 } VAEncFEIDistortionH264;    // 48 bytes
 
+/** \brief Motion Vector and Statistics frame level controls.
+ * VAStatsStatisticsParameterBufferType for H264 16x16 block
+ **/
+typedef struct _VAStatsStatisticsParameterH264
+{
+   VAStatsStatisticsParameter stats_params;
+
+    uint32_t    frame_qp                    : 8;
+    /** \brief maximum number of Search Units, valid range is [1, 63] */
+    uint32_t    len_sp                      : 8;
+    /** \brief motion search method definition
+     * 0: default value, diamond search
+     * 1: full search
+     * 2: diamond search
+     **/
+    uint32_t    search_path                 : 8;
+    uint32_t    reserved0                   : 8;
+
+    uint32_t    sub_mb_part_mask            : 7;
+    uint32_t    sub_pel_mode                : 2;
+    uint32_t    inter_sad                   : 2;
+    uint32_t    intra_sad                   : 2;
+    uint32_t    adaptive_search	            : 1;
+    /** \brief indicate if future or/and past MV in mv_predictor buffer is valid.
+     * 0: MV predictor disabled
+     * 1: MV predictor enabled for past reference
+     * 2: MV predictor enabled for future reference
+     * 3: MV predictor enabled for both past and future references
+     **/
+    uint32_t    mv_predictor_ctrl           : 3;
+    uint32_t    mb_qp                       : 1;
+    uint32_t    ft_enable                   : 1;
```

Comment:
What does ft stand for? Could you use a meaningful name?

Reply:
ft_enable means forward transform enable.